examples/webxr_ar_dragging.html

```diff
@@ -55,7 +54,7 @@
 				camera = new THREE.PerspectiveCamera( 50, window.innerWidth / window.innerHeight, 0.1, 10 );
 				camera.position.set( 0, 0, 3 );
 
-				controls = new OrbitControls( camera, container );
+				new OrbitControls( camera, container );
```

Comment:
Even if the linter complains here, I think it's more readable to write this line like so:
```js
const controls = new OrbitControls( camera, container );
```